common/Util-desktop.cpp

```diff
@@ -160,6 +162,56 @@ std::size_t getTotalSystemMemoryKb()
     return totalMemKb;
 }
 
+size_t getMaxConcurrentTCPConnections(const size_t minimum)
+{
+#ifdef __linux__
+    auto readDecimal = [](const char* path, const size_t defaultValue) -> size_t {
```

Comment:
Sigh - so - we can have a helper function for this - but lets name it something that is consistent with other Util helpers, and use similar code to implement it.

Reply:
This has been degraded to a local hidden lambda and we also do have `Util::dataToHexString` and `FileUtil::readFile` .. consistent verb `read` and the object to read `decimal`. 
Regardless, using `FileUtil::readFile` for the file ops as mentioned above, OK.